docs/source/i524/lectures-systems.csv

```diff
@@ -5,6 +5,6 @@ Virtualbox, Virtualbox for class, `Web page <..lesson/linux/virtualbox.html>`_,
 Shell, Linux Shell, `Video <https://www.youtube.com/watch?v=LeTlm_ck2GI>`_ | `Web Page <../lesson/linux/linux.html>`_,
 Python, (Draft) Introduction to Python, `Web page <../lesson/prg/python_intro.html>`_,
       , (Draft) Python for Big Data, `Web Page <../lesson/prg/python_big_data.html>`_,
-      , (Draft - Advanced) Python Fingerprint example, `Web page <../lesson/prg/python_lesson1.html>`_,
+      , ( - Advanced) Python Fingerprint example, `Web page <../lesson/prg/python_lesson1.html>`_,
```

Comment:
This was a test that should not be merged.
It looks like you are working on the wrong branch.